components/input-validation-mgt/org.wso2.carbon.identity.input.validation.mgt/src/main/java/org/wso2/carbon/identity/input/validation/mgt/model/validators/AbstractRulesValidator.java

```diff
@@ -126,6 +126,20 @@ protected boolean validatePositiveNumber(String value, String property, String t
         return Integer.parseInt(value) < 0;
     }
 
+    protected boolean validateBoolean(String value, String property, String tenantDomain)
```

Comment:
Can't we maintain the type of the value as a Boolean ?

Reply:
No, As per swagger definition, property values are always strings